src/lib/promiseDone.ts

```diff
@@ -0,0 +1,13 @@
+// @ts-ignore next-line
+if (typeof Promise.prototype.done !== "function") {
+  // @ts-ignore next-line
+  Promise.prototype.done = function (onFulfilled, onRejected) {
```

Comment:
Instead of patching could we [update to use `finally`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally)?

Reply:
Sadly not, they do fundamentally different things. The biggest being that `done` ends the promise chain and does not return a `Thenable`.

Reply:
Ah, damn 👍